misc/life_mngr/Makefile

```diff
@@ -0,0 +1,49 @@
+T := $(CURDIR)
+OUT_DIR ?= $(shell mkdir -p $(T)/build;cd $(T)/build;pwd)
+CC ?= gcc
+
+LIFEMNGR_CFLAGS := -g -O0 -std=gnu11
+LIFEMNGR_CFLAGS += -D_GNU_SOURCE
+LIFEMNGR_CFLAGS += -DNO_OPENSSL
+LIFEMNGR_CFLAGS += -m64
+LIFEMNGR_CFLAGS += -Wall -ffunction-sections
+LIFEMNGR_CFLAGS += -Werror
+LIFEMNGR_CFLAGS += -O2 -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=2
+LIFEMNGR_CFLAGS += -Wformat -Wformat-security -fno-strict-aliasing
+LIFEMNGR_CFLAGS += -fpie -fpic
+LIFEMNGR_CFLAGS += $(CFLAGS)
+
+GCC_MAJOR=$(shell echo __GNUC__ | $(CC) -E -x c - | tail -n 1)
+GCC_MINOR=$(shell echo __GNUC_MINOR__ | $(CC) -E -x c - | tail -n 1)
+
+#enable stack overflow check
+STACK_PROTECTOR := 1
+
+ifdef STACK_PROTECTOR
+ifeq (true, $(shell [ $(GCC_MAJOR) -gt 4 ] && echo true))
+LIFEMNGR_CFLAGS += -fstack-protector-strong
+else
+ifeq (true, $(shell [ $(GCC_MAJOR) -eq 4 ] && [ $(GCC_MINOR) -ge 9 ] && echo true))
+LIFEMNGR_CFLAGS += -fstack-protector-strong
+else
+LIFEMNGR_CFLAGS += -fstack-protector
+endif
+endif
+endif
+
+LIFEMNGR_LDFLAGS := -Wl,-z,noexecstack
+LIFEMNGR_LDFLAGS += -Wl,-z,relro,-z,now
+LIFEMNGR_LDFLAGS += -pie
+LIFEMNGR_LDFLAGS += $(LDFLAGS)
+
+all:
```

Comment:
need target install in this Makefile?

Reply:
lifemngr is running on UOS, not on SOS.
So it no need to install.